doc/main/tbb_userguide/Floating_Point_Settings.rst

```diff
@@ -32,19 +31,19 @@ You can then pass the task group context to most parallel algorithms, including
 It is possible to execute the parallel algorithms with different floating-point settings captured to separate contexts, even at the same time.
 
 Floating-point settings captured to a task group context prevail over the settings captured during task scheduler initialization. It means, if a context is passed to a parallel algorithm, the floating-point settings captured to the context are used. 
-Otherwise, if floating-point settings are not captured to the context, or a context is not explicitly specified, the settings captured during task scheduler initialization are used.
+Otherwise, if floating-point settings are not captured to the context, or a context is not explicitly specified, the settings captured during the task arena initialization are used.
 
 In a nested call to a parallel algorithm that does not use the context of a task group with explicitly captured floating-point settings, the outer-level settings are used. 
 If none of the outer-level contexts capture floating-point settings, the settings captured during task scheduler initialization are used.
```

Comment:
Change here the "task scheduler" to the "task arena" as well, similarly to line 34